lib/tvdbapiv2/api_client.py

```diff
@@ -67,52 +61,36 @@ class ApiClient(object):
     :param header_name: a header to pass when making calls to the API.
     :param header_value: a header value to pass when making calls to the API.
     """
-    def __init__(self, host=None, cookie=None, session=None, user_agent=None, api_key=None):
+    def __init__(self, host=None, cookie=None, session=None, api_key=None, token=None):
 
         """
         Constructor of the class.
         """
         self.session = session or requests.session()
-        if api_key:
-            self.session.auth = TVDBAuth(api_key)
+        if not api_key and not token:
```

Comment:
`if not (api_key or token)`  It only requires one or the other, not both.  The purpose of the token is if you already have a previous token-auth you wish to reuse.